sql/src/test/java/io/crate/metadata/doc/DocIndexMetaDataTest.java

```diff
@@ -81,7 +81,7 @@ private DocIndexMetaData newMeta(IndexMetaData metaData, String name) throws IOE
     }
 
     @Before
-    public void before() throws Exception {
+    public void beforeTest() throws Exception {
```

Comment:
just rename it to what the method is really doing is much better and avoids conflicts/overrrides